app/controllers/user/profile/orders_controller.rb

```diff
@@ -0,0 +1,6 @@
+class User::Profile::OrdersController < User::BaseController
+	
+	def index
+		@orders = Order.where(user: current_user)
+	end
```

Comment:
This is awesome and will be insanely useful for my user stories. Thanks for setting this one up!